src/attr.rs

```diff
@@ -6,13 +6,97 @@ use std::iter;
 #[derive(Debug, Clone, Eq, PartialEq, Hash)]
 pub struct Attribute {
     pub style: AttrStyle,
-    pub value: MetaItem,
+    pub name: Ident,
```

Comment:
The one in rustc supports any Path here - for example they have this unit test:

```rust
#[::attr_args::identity
  fn main() { assert_eq!(foo(), "Hello, world!"); }]
struct Dummy;
```

Any idea how much harder that would be to support?